### PR TITLE
Remove Godeps, update Travis to use glide

### PR DESCRIPTION
**Pull request by prashantv**
Change travis to target go1.5 or higher since 1.4 doesn't support vendor

r @sectioneight

### CODE REVIEW

**Comment by sectioneight** [comment]
Awesome. :heart: glide, :shipit:

**Comment by sectioneight** [comment]
Also you will probably (?) need to change our internal debian job, lmk if you want help with that

File: .travis.yml [hunk 1]
```diff
@@ -2,13 +2,14 @@
 language: go
 
 go:
-  - 1.4
+  - 1.5
```
Comment (sectioneight) [inline .travis.yml]: maybe a comment about how 1.4 is **probably** supported, we just don't test it because glied?

**Comment by sectioneight** [comment]
Also, need to update the readme:

> Install the Go dependencies:
> 
> $ go get github.com/tools/godep
> $ godep restore

File: .travis.yml (hunk 1 above)
Comment (prashantv) [inline .travis.yml]: good idea, done

**Comment by prashantv** [comment]
Updated README to use `glide` and also added instructions on how `docker run` can be used.

**Comment by sectioneight** [comment]
doop looks great you are great

**Pull request merged**